packages/devtools_app/lib/src/inspector/inspector_tree.dart

```diff
@@ -218,7 +218,7 @@ class InspectorTreeNode {
           ticks: ticks,
           depth: depth,
           lineToParent:
-              !node.isProperty && index != 0 && node.parent.showLinesToChildren,
+          !node.isProperty && index != 0 && node.parent.showLinesToChildren,
```

Comment:
I think formatting is off here. Maybe make sure you are on the latest flutter and then run `flutter/bin/dart format .`

Reply:
fixed